invenio_app_ils/ill/schemas/ill_borrowing_requests/borrowing_request-v1.0.0.json

```diff
@@ -54,13 +54,17 @@
   },
   "properties": {
     "$schema": {
-      "title": "Interlibrary borrowing request schema",
+      "title": "InterLibrary borrowing request schema",
       "type": "string"
     },
     "cancel_reason": {
       "title": "Reason for cancelling the ILL borrowing request",
       "type": "string"
     },
+    "created_by_pid": {
```

Comment:
I know that it is not the point of the PR, but as a suggestion, to unify this.
it is not possible that the borrowing request gets created by a bot, right?
Anyway, we have the `created_by` in the document, but there it might be also a harvest potentially. Why not go with (in every case where we need it):
```
created_by: {
  scheme: "user",
  value: "3"
}
```
it would be more flexible.

Reply:
Good point. I honestly copy the `created_by_pid` field, but I will change it to something like this in the next PR! :)